osu.Game/Skinning/Editor/SkinBlueprintContainer.cs

```diff
@@ -90,6 +93,47 @@ protected override void AddBlueprintFor(ISkinnableDrawable item)
             base.AddBlueprintFor(item);
         }
 
+        protected override bool OnKeyDown(KeyDownEvent e)
+        {
+            switch (e.Key)
+            {
+                case Key.Left:
+                    moveSelection(new Vector2(-1, 0));
+                    return true;
+
+                case Key.Right:
+                    moveSelection(new Vector2(1, 0));
+                    return true;
+
+                case Key.Up:
+                    moveSelection(new Vector2(0, -1));
+                    return true;
+
+                case Key.Down:
+                    moveSelection(new Vector2(0, 1));
+                    return true;
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Move the current selection spatially by the specified delta, in screen coordinates (ie. the same coordinates as the blueprints).
+        /// </summary>
+        /// <param name="delta"></param>
+        private void moveSelection(Vector2 delta)
+        {
+            var firstBlueprint = SelectionHandler.SelectedBlueprints.FirstOrDefault();
+
+            if (firstBlueprint == null)
+                return;
+
+            // convert to game space coordinates
+            delta = firstBlueprint.ToScreenSpace(delta) - firstBlueprint.ToScreenSpace(Vector2.Zero);
```

Comment:
not 100% on working in screen coordinates for skins (they will be varying in accuracy depending on the client resolution) but i guess it's fine for now. the path forward is likely to have a grid for skinning which brings some sensibility to positioning.